### PR TITLE
fix: add/fix text blocks' language

### PR DESCRIPTION
**Pull request by waynexia**
Signed-off-by: Ruihang Xia <waynestxia@gmail.com>

### CODE REVIEW

File: docs/user-guide/supported-protocols/influxdb.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ monitor,host=host4 cpu=66.9,memory=1027 1663840496400340000'
 
 We can always query the metrics with SQL:
 
-```text
+```sql
 mysql> SELECT * FROM influxdb.monitor;
 +-------+------+--------+---------------------+
```
Comment (killme2008) [inline docs/user-guide/supported-protocols/influxdb.md:40]: Remove the `mysql` prefix and move out the output to an independent text block. Then user can copy this SQL for test.

File: docs/user-guide/supported-protocols/opentsdb.md [hunk 2]
```diff
@@ -118,7 +118,7 @@ curl -X POST http://127.0.0.1:4000/v1/opentsdb/api/put -d '
 
 You can always query the metrics with SQL:
 
-```text
+```sql
```
Comment (killme2008) [inline docs/user-guide/supported-protocols/opentsdb.md:121]: The same with above

File: docs/user-guide/table-management.md [hunk 3]
```diff
@@ -17,7 +17,7 @@ mysql>
 
 In this example, we are going to create a table named `monitor`
 
-``` shell
+``` sql
```
Comment (killme2008) [inline docs/user-guide/table-management.md:20]: The same with above

File: docs/user-guide/table-management.md [hunk 4]
```diff
@@ -46,7 +46,7 @@ to tags in other time-series systems like [InfluxDB][1].
 
 You can use `show tables` statement to list existing tables
 
-``` shell
+``` sql
```
Comment (killme2008) [inline docs/user-guide/table-management.md:49]: The same with above

File: docs/user-guide/table-management.md [hunk 5]
```diff
@@ -60,7 +60,7 @@ mysql> show tables;
 Notice: `script` table is a built-in table that holds User-Defined Functions (UDFs).
 Currently only table name filtering is supported. You can filter existing tables by their names.
 
-``` shell
+``` sql
```
Comment (killme2008) [inline docs/user-guide/table-management.md:63]: The same with above

File: docs/user-guide/supported-protocols/http-api.md [hunk 6]
```diff
@@ -166,12 +166,12 @@ def square(number):
 
 Submits it to database:
 
-```
+```shell
```
Comment (zyy17) [inline docs/user-guide/supported-protocols/http-api.md:169]: nit: use `console` will be better display.

**Comment by waynexia** [comment]
>Cloudflare Pages Build failed

I'll remove `proto` marker to make it build

**Comment by waynexia** [comment]
> > Cloudflare Pages Build failed
> 
> I'll remove `proto` marker to make it build

It still fails... @sunng87 could you please help to grab the error message? I don't have that account.

**Comment by xtang** [comment]
<pre class="c_ni c_bq c_ml c_mm c_qz c_ra c_da c_rq c_iq c_io c_ip c_ev c_ew c_rb c_bs c_rr">

11:46:09.101 | file: /opt/buildhome/repo/docs/developer-guide/get-started.md
-- | --
11:46:09.101 | ✖ building client + server bundles...
11:46:09.109 | build error:
11:46:09.109 | Error: No language registration for console
11:46:09.109 | at getGrammar (/opt/buildhome/repo/node_modules/vitepress/node_modules/shiki/dist/index.js:2213:13)
11:46:09.110 | at codeToThemedTokens (/opt/buildhome/repo/node_modules/vitepress/node_modules/shiki/dist/index.js:2222:26)
11:46:09.110 | at Object.codeToHtml (/opt/buildhome/repo/node_modules/vitepress/node_modules/shiki/dist/index.js:2236:20)
11:46:09.110 | at Object.codeToHtml (file:///opt/buildhome/repo/node_modules/vitepress/dist/node/serve-fcdc8868.js:34892:39)
11:46:09.110 | at Object.highlight (file:///opt/buildhome/repo/node_modules/vitepress/dist/node/serve-fcdc8868.js:34954:21)
11:46:09.110 | at default_rules.fence (file:///opt/buildhome/repo/node_modules/vitepress/dist/node/serve-fcdc8868.js:18041:27)
11:46:09.110 | at md.renderer.rules.fence (file:///opt/buildhome/repo/node_modules/vitepress/dist/node/serve-fcdc8868.js:34998:16)
11:46:09.110 | at md.renderer.rules.fence (file:///opt/buildhome/repo/node_modules/vitepress/dist/node/serve-fcdc8868.js:35329:21)
11:46:09.110 | at Object.md.renderer.rules.fence (file:///opt/buildhome/repo/node_modules/vitepress/dist/node/serve-fcdc8868.js:35319:12)
11:46:09.110 | at Renderer$1.render (file:///opt/buildhome/repo/node_modules/vitepress/dist/node/serve-fcdc8868.js:18320:38) {
11:46:09.111 | code: 'PLUGIN_ERROR',
11:46:09.111 | plugin: 'vitepress',
11:46:09.111 | hook: 'transform',
11:46:09.111 | id: '/opt/buildhome/repo/docs/developer-guide/get-started.md',
11:46:09.111 |  
</pre>
@waynexia FYI

**Review by killme2008 — APPROVED** [review]
LGTM

**Pull request merged**